### PR TITLE
Add forklifts, barriers and cages

### PR DESCRIPTION
**Pull request by c00kiepreferences**
## Pull Request details
Added two Gems with assets:
- `RobotecWarehouseBarriersAndCagesAssets`: Gem with warehouse barriers and cage walls assets.
- `RobotecWarehouseForkliftsAssets`: Gem with warehouse forklifts assets.

## Checklist
- [x] I updated the `README.md` document to add information about new Gem(s).
- [x] I added git attributes file.
- [x] I filled in the `gem.json` file with the necessary information, including license, origin, and summary in the new Gem(s).
- [x] I checked if the folder structure and filenames follow the convention described in `README.md`.
- [x] I added the `NOTICE` file with some additional comments on the license. 
- [x] I checked if the origins of prefabs follow the convention.

### CODE REVIEW

**Review by jhanca-robotecai — CHANGES_REQUESTED** [review]
I am okay with the names and the structure, it follows the proposed scheme. The only problem I see is that LFS is not properly initialized. The PRs need to be reordered.

**Review by jhanca-robotecai — APPROVED** [review]
I fixed the LFS and force-pushed the code. It is okay for me know.

File: RobotecWarehouseBarriersAndCagesAssets/Assets/Cages/Models/Materials/Textures/Cage-BaseColor.png
Comment (pawelbudziszewski) [inline RobotecWarehouseBarriersAndCagesAssets/Assets/Cages/Models/Materials/Textures/Cage-BaseColor.png:1]: Names of these files should be separated by an underscore, not a dash

**Review by pawelbudziszewski — CHANGES_REQUESTED** [review]
Forklifts should be rotated to have X-axis forward.

File: RobotecWarehouseBarriersAndCagesAssets/Assets/Cages/Models/Materials/Textures/Cage-BaseColor.png
Comment (c00kiepreferences) [reply RobotecWarehouseBarriersAndCagesAssets/Assets/Cages/Models/Materials/Textures/Cage-BaseColor.png:1]: I tried to change that, but for some reason, as soon as I changed the dash to the underscore, textures became broken. Other names i tried did not show this kind of issue (i also checked with asset processor, it wasn't a case of it still processing textures).

Comment (jhanca-robotecai) [reply RobotecWarehouseBarriersAndCagesAssets/Assets/Cages/Models/Materials/Textures/Cage-BaseColor.png:1]: PNG files with `_BaseColor` suffix are automatically loaded as `Albedo`. I added `assetinfo` files to overwrite this issue.

**Review by pawelbudziszewski — APPROVED** [review]

**Pull request merged**